experimental/devices/mfrc522/example_test.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that can be found in the LICENSE file.
+
+package mfrc522_test
+
+import (
+	"encoding/hex"
+	"log"
+	"reflect"
+	"time"
+
+	"periph.io/x/periph/conn/spi/spireg"
+	"periph.io/x/periph/experimental/devices/mfrc522"
+	"periph.io/x/periph/experimental/devices/mfrc522/commands"
+	"periph.io/x/periph/host"
+	"periph.io/x/periph/host/rpi"
+)
+
+func Example() {
+	// Make sure periph is initialized.
+	if _, err := host.Init(); err != nil {
+		log.Fatal(err)
+	}
+
+	// Using SPI as an example. See package ./spi/spireg for more details.
+	p, err := spireg.Open("")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer p.Close()
+
+	rfid, err := mfrc522.NewSPI(p, rpi.P1_13, rpi.P1_11)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	// Idling device on exit.
+	defer rfid.Halt()
+
+	// Setting the antenna signal strength.
+	rfid.SetAntennaGain(5)
```

Comment:
Please check for error everywhere, so that people can copy paste the code safely and this encourages good code.